frontend/src/app/modules/settings/team/team-members/team-members.component.ts

```diff
@@ -161,14 +164,27 @@ export class TeamMembersComponent extends BaseComponent implements OnInit, OnDes
             );
     }
 
-    changeTeamMemberRole(user: IUser, newRole: Role) {
+    changeTeamMemberRole(user: ITeamMember, event: MatSelectChange) {
+        const selectedRole: Role = Role[event.value as keyof typeof Role];
+
+        event.source.value = user.role;
+        if (selectedRole === Role.Owner && this.ownerId) {
+            this.notificationService.showErrorMessage('Team member with owner role should be the only');
+
+            return;
+        }
+        if (user.id === this.ownerId) {
+            this.notificationService.showErrorMessage('Owner role can be modified only when owner leave the team');
+
+            return;
+        }
```

Comment:
Why we cannot just disable dropdown?

Reply:
According to QA feedback, only dropdowns disabling isn`t enough